sdk/python/arvados/keep.py

```diff
@@ -22,6 +22,16 @@
 global_client_object = None
 
 
+# Monkey patch TCP constants when not available (apple). Values sourced from:
+# http://www.opensource.apple.com/source/xnu/xnu-2422.115.4/bsd/netinet/tcp.h
+if not hasattr(socket, 'TCP_KEEPALIVE'):
+    socket.TCP_KEEPALIVE = 0x010
+if not hasattr(socket, 'TCP_KEEPINTVL'):
+    socket.TCP_KEEPINTVL = 0x101
+if not hasattr(socket, 'TCP_KEEPCNT'):
+    socket.TCP_KEEPCNT = 0x102
```

Comment:
I don't think we can assume OSX is the only platform where those constants aren't already set. How about putting this block in an `if sys.platform == 'darwin':`?